src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -23,6 +23,37 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         {
         }
 
+        [Theory]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", false)]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", true)]
+        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, string property, bool isEnabled)
+        {
+            var asset = _testAssetsManager
+                .CopyTestAsset(testAssetName, identifier: "C#")
```

Comment:
The identifier should be composed of the test parameters that differ between different Theory instances.

```suggestion
                .CopyTestAsset(testAssetName, identifier: isEnabled.ToString())
```